### PR TITLE
Update process

### PR DESCRIPTION
**Pull request by barath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,134 +1,122 @@
...
 
-Anyone may participate in [this process](https://github.com/solid/culture). Please read the [Code of Conduct](code-of-conduct.md) before doing so.
+Anyone may participate in this process. Please read the [Code of Conduct](code-of-conduct.md) before doing so.
+
+## Editorial Team
+
+The Solid [Editorial Team](https://github.com/orgs/solid/teams/editors) is responsible for the implementation of the Solid Specification process. Members of the Editorial Team are appointed by the Solid Director. The Editorial Team is comprised of all the Editors appointed by the Solid Director, who are listed at [editors.md](editors.md), along with their assignments, contact details, and affiliations. The Editorial Team shepherds Solid development efforts and coordinates with those who participate in Solid Panels and with Stakeholders and Implementers. The Solid Director may also appoint support personnel on an as-needed basis, such as a Release Manager, for organizing the workflow of the specification process.
```
Comment (oolivo) [inline README.md]: "The Editorial team shepherds  Solid Specification authoring efforts". Development might be misconstrued in some way as it's a bit vague.

File: README.md [hunk 2]
```diff
@@ -1,134 +1,122 @@
...
 ## Contributors
 
-Any individual that has been involved in proposals to improve the [Solid Specification](https://github.com/solid/specification) has provided a valuable service to the [Solid Project](https://www.solidproject.org) and is encouraged to continue.
+Any individual who has been involved in proposals to improve the [Solid Specification](https://github.com/solid/specification) has provided a valuable service to the [Solid Project](https://www.solidproject.org) and is encouraged to continue.
 
-All manner of contributions are important - whether identifying problems, asking questions, or proposing normative changes.
+Contributions are welcome, including identifying problems, asking questions, and proposing normative changes.
 
-There are many topics, problems, or ideas best tackled by a group of people with a specific set of domain expertise. For these cases, we have [Solid Panels](#solid-panels).
+There are many topics, problems, or ideas best addressed by a group of people with specific domain expertise in [Solid Panels](#solid-panels).
```
Comment (oolivo) [inline README.md]: %s/people/contributors/

File: README.md [hunk 3]
```diff
@@ -1,134 +1,122 @@
...
 
-Solid Panels are groups of individuals focused on a specific problem or domain relevant to Solid, with an aim to propose changes to the [Solid Specification](https://github.com/solid/specification). Domains may be technical, non-technical, or some combination of the two. For example, a Security Panel could focus on the evaluation and advancement of the Solid security model. Only members of the [W3C Solid Community Group](https://www.w3.org/community/solid/), which anyone may join, and which operates under the [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/), may become a member of a Solid Panel.
+Solid Panels are groups of individuals focused on a specific problem or domain relevant to Solid, with an aim to propose changes to the [Solid Specification](https://github.com/solid/specification). Domains may be technical, non-technical, or some combination of the two. For example, a Security Panel could focus on the evaluation and advancement of the Solid security model. Only those who have agreed to the [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/) may participate in a Solid Panel.
```
Comment (oolivo) [inline README.md]: %s/individuals/contributors/g

File: README.md [hunk 4]
```diff
@@ -1,134 +1,122 @@
...
+New panels may be proposed by submitting an issue to the [solid/process](https://github.com/solid/process) repository. Establishing a new panel requires the endorsement of at least one member of the Editorial Team. To receive endorsement, the proposed panel submission must include:
 
 1. A stated purpose
 1. One or more initiatives that will be actively pursued and tracked in regular panel sessions to advance the stated purpose
-1. Demonstrable support of at least five prospective members
+1. Demonstrable support of at least five prospective participants
 
-Any iteration on the stated purpose, initiatives, and supporting members should continue as needed until endorsement is received from at least one member of the Editorial Team. Submissions that fail to receive endorsement may be removed by Solid Administrators after six months.
+The panel proposal can be revised until at least one member of the Editorial Team endorses the proposal. Submissions that fail to receive endorsement may be removed by Solid Administrators after six months.
```
Comment (oolivo) [inline README.md]: Let's move quicker to have less noise lingering around. 90 days?

File: README.md [hunk 5]
```diff
@@ -1,134 +1,122 @@
...
 
 Stakeholders who have opted to identify themselves publicly are listed at [stakeholders.md](stakeholders.md). Anyone may decide to identify themselves publicly as a Solid Stakeholder, but it is not mandatory. Identified stakeholders may be consulted for feedback as part of the editorial process.
 
-### Solid Users
-
-Solid Users are individuals, companies, or organizations who access data stored in a Solid Pod.
-
 ### Implementers
-Solid Implementers are companies or organizations who are implementing the [Solid Specification](https://github.com/solid/specification). A Solid Implementer may be any combination of Identity Provider, Pod Provider, and Application Provider.
+Solid Implementers are organizations who are implementing the [Solid Specification](https://github.com/solid/specification). A Solid Implementer may be any combination of Identity Provider, Pod Provider, and Application Provider.
```
Comment (oolivo) [inline README.md]: Do identity providers implement the solid spec? Do client libraries fall under Application provider?

File: README.md [hunk 6]
```diff
@@ -1,134 +1,122 @@
...
 
 To help broad consensus form, it is suggested that each proposal be brought to the attention of the Editors at some defined contact points during its life cycle. These contact points are:
-1. When a reason to change or extend the specification is first encountered, an [issue is opened](https://github.com/solid/specification/issues/new). The issue is then used for further communication with the Editors.
+1. When a reason to change or extend the specification is first encountered, an [issue can be created](https://github.com/solid/specification/issues/new). The issue can then be used for further communication with the Editors.
 1. Panels may create or adopt issues, and Panels should notify Editors that such discussion has started.
 1. When a Panel reaches rough consensus on an issue, the Panel should notify the Editors, indicating in informal text what the consensus entails and who participated in forming it.
 1. When a Panel starts drafting the text of a proposal, the Panel should notify the Editors.
```
Comment (oolivo) [inline README.md:98]: How does one notify the editors?

File: README.md [hunk 7]
```diff
@@ -1,134 +1,122 @@
...
+Candidate Proposals with substantive changes must be responsive to comments and objections. If rough consensus cannot be reached among the Solid Editors, a vote may be taken during a meeting of the Editors, and the advancement of the proposal can be blocked by a negative vote by 2/3 of the Editors. Once the Candidate Proposal has successfully passed editorial review, it may be merged by an assigned Editor. The shepherd for a proposal is responsible for ensuring a resolution of the proposal within the time window, and if the shepherd is unable to complete the proposal review during the time window, the Solid Director will appoint a different Editor as shepherd and extend the time window for the proposal by 1 month.
```
Comment (oolivo) [inline README.md]: %s/may/must

File: README.md [hunk 8]
```diff
@@ -1,134 +1,122 @@
...
+When a proposal is first submitted, the Solid Editors will assign a single Editor as the shepherd for the proposal and assign a time window for review and its acceptance or rejection; the time window will be either 1 or 2 months from the date of proposal submission, depending on the complexity of the proposal. This time window includes all discussion and revision that may be needed to improve the proposal. The Solid Director, working with the Solid Editors and any administrative support personnel, will maintain a dashboard of open and closed proposals, time windows for review, and review status.
+
+If an Editor or other person disapproves of Candidate Proposal that is under consideration, they must provide substantive, written comments that contain proposed remedies to the Solid Editors at least 1 week before the end of the review time window. The Solid Editors will consider the comments and suggest possible remedies.
```
Comment (oolivo) [inline README.md]: %s/Editor or other person/Contributor/

File: README.md (hunk 7 above)
Comment (oolivo) [inline README.md]: If a proposal is rejected/sent back for rework. Are there any limits on when it can be resubmitted?

File: README.md [hunk 9]
```diff
@@ -1,134 +1,122 @@
...
+Editors have [_Admin Permissions_](https://help.github.com/en/articles/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization) on the repositories they are assigned to, and are permitted to grant [_Write Permissions_](https://help.github.com/en/articles/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization) to other contributing authors on the same. All Editors have [_Write Permissions_](https://help.github.com/en/articles/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization) on all repositories requiring editorial review listed in [editors.md](editors.md).
 
 # Test Suite
 Test Suite Developers work with the aim is to develop a [test suite](https://github.com/solid/test-suite) that can be used to verify an implementation against the Solid specification.
```
Comment (oolivo) [inline README.md]: @barath @emmettownsend 

Why is this here? The independent test suite is not officially endorsed over any other test suite by the Spec or Solid team. Who added this in here in the first place? Let's remove it.

**Comment by oolivo** [comment]
I'm also realizing this document links to various other documents that are in rough shape. 

panels.md needs to be updated to include panel chairs, editors.md might need some work too.

File: README.md (hunk 5 above)
Comment (barath) [reply README.md]: True, identity providers may not be implementers of the solid spec. As for client libraries, I think they do fall under application development, though perhaps not "Provider"

File: README.md (hunk 6 above)
Comment (barath) [reply README.md:98]: I assume through github tags? I didn't want to change the old text, which was unspecific in this regard.

File: README.md (hunk 9 above)
Comment (barath) [reply README.md]: I'm happy to delete it; I don't know why it was originally here.

File: README.md (hunk 7 above)
Comment (barath) [reply README.md]: I made some tweaks to this paragraph to clarify; see what you think.

**Review by oolivo — APPROVED** [review]

File: README.md [hunk 10]
```diff
@@ -1,137 +1,123 @@
 This repository details how changes to Solid may be proposed and accepted.
 
 # Solid Specification
-The following is a description about how changes to the specifications in the Solid ecosystem may be proposed and accepted.
+The following describes how changes to the specifications in the Solid ecosystem may be proposed and accepted. Anyone may submit a proposal to alter this process; such proposals will be reviewed only by [Tim Berners-Lee](https://github.com/timbl), who is the Solid Director.
 
-The current development on the Solid ecosystem is carried out in the [Solid Specifications](https://github.com/solid/specification) repository. The [Unofficial Draft](https://github.com/solid/solid-spec) predating the current development is still available for historical reference.
+Development in the Solid ecosystem occurs in the [Solid Specifications](https://github.com/solid/specification) repository. The [Unofficial Draft](https://github.com/solid/solid-spec) predating current development is still available for historical reference.
```
Comment (emmettownsend) [inline README.md]: Is it worth saying that the unofficial draft is deprecated? If you go to that repo is says :

**This document contains an informal description of implementation guidelines for Solid servers and clients. A normative specification is in the making at https://github.com/solid/specification/[](https://github.com/solid/solid-spec#table-of-contents). For the time being, the present document contains the best approximation of expected server and client behavior.**

I think that gives a misleading impression. The current version should now be the version that was released some weeks ago.

File: README.md [hunk 11]
```diff
@@ -1,137 +1,123 @@
...
 
-Anyone may participate in [this process](https://github.com/solid/culture). Please read the [Code of Conduct](code-of-conduct.md) before doing so.
+Anyone may participate in this process. Please read the [Code of Conduct](code-of-conduct.md) before doing so.
+
+## Editorial Team
+
+The Solid [Editorial Team](https://github.com/orgs/solid/teams/editors) is responsible for the implementation of the Solid Specification process. Members of the Editorial Team are appointed by the Solid Director. The Editorial Team is comprised of all the Editors appointed by the Solid Director, who are listed at [editors.md](editors.md), along with their assignments, contact details, and affiliations. The Editorial Team shepherds Solid and coordinates with those who participate in Solid Panels and with Stakeholders and Implementers. The Solid Director may also appoint support personnel on an as-needed basis, such as a Release Manager, for organizing the workflow of the specification process.
```
Comment (emmettownsend) [inline README.md]: The assignments at editors.md do not seem to make sense.
Some have no primary documents. 
Some are not being worked on at all
Some people are on way too many assignments
Its not clear who the accountable person is for each one.

Comment (emmettownsend) [inline README.md]: Are Solid Panels, Stakeholders and Implementers defined somewhere? If so we should link to them in this paragraph. 

I'm not sure what it means when we say 'The Editorial Team shepherds Solid.." Do we mean the process or the deliverables?

File: README.md [hunk 12]
```diff
@@ -1,137 +1,123 @@
...
+Solid Panels are groups of contributors focused on a specific problem or domain relevant to Solid, with an aim to propose changes to the [Solid Specification](https://github.com/solid/specification). Domains may be technical, non-technical, or some combination of the two. For example, a Security Panel could focus on the evaluation and advancement of the Solid security model. Only those who have agreed to the [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/) may participate in a Solid Panel.
 
-New panels may be proposed by submitting an issue to the [solid/process](https://github.com/solid/process) repository. Establishing a new panel requires the endorsement of at least one member of the [Editorial Team](https://github.com/orgs/solid/teams/editors). To receive endorsement, the proposed panel submission must include:
+New panels may be proposed by submitting an issue to the [solid/process](https://github.com/solid/process) repository. Establishing a new panel requires the endorsement of at least one member of the Editorial Team. To receive endorsement, the proposed panel submission must include:
```
Comment (emmettownsend) [inline README.md:28]: Should this not be a majority? Otherwise there are likely to be multiple panels with no activity; in fact that already happens. If there is to be focused work that achieves outcomes it feels appropriate that a majority of the editors would agree that this work needs to be done. 
We have examples of panels that are doing loads of work but where most of the editors are not engaged. How will that work make its way into the spec?

File: README.md [hunk 13]
```diff
@@ -1,137 +1,123 @@
...
+- Working with Editors on prioritizing panel initiatives
 - Ensuring panel meetings are focused on advancing panel initiatives
-- Ensuring every Panel meeting has a clear goal and agenda
+- Ensuring every panel meeting has a clear goal and agenda
 - Ensuring all actions arising from meetings are tracked to completion
-- Ensuring target dates are set for outcomes
-- Working with Panel members to achieve broad consensus on decisions, with a consistent aim towards unanimity, in general accordance with the W3C process for realizing [consensus](https://www.w3.org/2019/Process-20190301/#Consensus).
-- Referring to the W3C process to [constructively manage dissent](https://www.w3.org/2019/Process-20190301/#managing-dissent) from panel members so that the group can continue to make progress in the absence of broad consensus.
-- Ensuring panel decisions, activities, and achievements are tracked and communicated.
+- Ensuring target dates are set for objectives
```
Comment (emmettownsend) [inline README.md:47]: If actions from meetings are rarely carried out and if target dates for objectives are never met, what happens then?
Should a panel be dissolved if it does not make progress after a given period of time? I know if there is no activity then it can be removed but no progress is almost worse as it means there is work to be done and the group involved is not getting it done?

What about panels that started with 5 or 6 people but quickly get to a point where only a couple of people attend  or do any work? Should that panel continue? Does it indicate there is not sufficient interest in the topic of discussion?

File: README.md [hunk 14]
```diff
@@ -1,137 +1,123 @@
...
 Stakeholders who have opted to identify themselves publicly are listed at [stakeholders.md](stakeholders.md). Anyone may decide to identify themselves publicly as a Solid Stakeholder, but it is not mandatory. Identified stakeholders may be consulted for feedback as part of the editorial process.
 
-### Solid Users
-
-Solid Users are individuals, companies, or organizations who access data stored in a Solid Pod.
-
 ### Implementers
-Solid Implementers are companies or organizations who are implementing the [Solid Specification](https://github.com/solid/specification). A Solid Implementer may be any combination of Identity Provider, Pod Provider, and Application Provider.
+
+Solid Implementers are organizations who are implementing the [Solid Specification](https://github.com/solid/specification). A Solid Implementer may be any combination of Identity Provider, Pod Provider, Application Provider, or a provider of other Solid-related functionality.
```
Comment (emmettownsend) [inline README.md:68]: Should each panel have at least one Implementer as a member?

**Review by emmettownsend — COMMENTED** [review]
I didn't see anything in there about the process to determine what goes into a specific version of the spec(s). Is that somewhere else?

File: README.md (hunk 11 above)
Comment (barath) [reply README.md]: Might we want to ask them to clean up that assignment / allocation after we update the process?

File: README.md (hunk 10 above)
Comment (barath) [reply README.md]: Good point, updated.

File: README.md (hunk 11 above)
Comment (barath) [reply README.md]: Linked. And the comment about the editorial team is intentionally vague so that they don't overstep their bounds but also so it's not too prescriptive.

File: README.md (hunk 12 above)
Comment (barath) [reply README.md:28]: This process I hope makes the panel a bit more advisory and less formally required in the process, since I've removed the option for a panel to vote on anything. As a result, a panel is just a loose working group, and as such there's no need even for a panel to be endorsed (but we can allow for it with the low bar of a single editor agreeing to it). Anyone can propose spec changes, either based upon panel work or outside work.

File: README.md (hunk 13 above)
Comment (barath) [reply README.md:47]: Perhaps, but I'm less worried about garbage collecting panels if they are non-formal groups anyway. We create them here, but they are given basically no powers / authority. So if they stagnate, there's no harm, though it would be good to remove them. I've added a line that says that the editors can close a panel due to inactivity by a majority vote of the editors.

File: README.md (hunk 14 above)
Comment (barath) [reply README.md:68]: I think this might be difficult to enforce, since panels don't have formal membership in this model.
